tickets/src/components/interface/EventLinks.js

```diff
@@ -5,7 +5,7 @@ import { googleCalendarLinkBuilder } from '../../utils/links'
 
 export const EventLinks = ({ event }) => {
   if (typeof window === 'undefined') {
-    return <></>
+    return <></> // WHAT IS THIS???
```

Comment:
https://reactjs.org/docs/fragments.html#short-syntax

Reply:
Ho, TIL!

Reply:
In this case it's just being used to return "empty" JSX because that function has a dependency on `window` that may not be there (IIRC I added it because I got SSR errors a few times). I think in the future we'll be able to remove it, just at the moment that component is a verbatim copy/paste from the megafunction that is `EventContent`